src/rules/matchDescription.js

```diff
@@ -53,9 +53,19 @@ export default iterateJsdoc(({
     return;
   }
 
+  const hasOptionTag = (tagName) => {
+    return {}.hasOwnProperty.call(options.tags, tagName) && options.tags[tagName];
```

Comment:
```suggestion
    return !!options.tags[tagName];
```
The first part of the condition seems to be redundant.

Reply:
While it's pretty obscure and not needed here, I've gotten in the habit since Firefox (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Undefined_prop ) will, if its `about:config` setting `javascript.options.strict` is enabled (unrelated to "strict mode"), report missing property references as warnings in console, whereas the `hasOwnProperty` check will avoid this. But I can remove the check as it is indeed not really needed (unless someone is adding stuff on `Object.prototype`).

Reply:
I've adjusted per your suggestion, though using `Boolean()` to coerce instead of `!!` per the linting rules.